plugins/datahub-integration/src/main/java/org/fao/geonet/datahub/DatahubController.java

```diff
@@ -49,6 +49,12 @@ public ResponseEntity<String> getDatahubStatus() throws IOException {
         String defaultConfig = FileUtils.readFromInputStream(new FileInputStream(configFile));
         JSONObject body = new JSONObject();
         body.put("defaultConfig", defaultConfig);
+
+        File packageJsonFile = new File("web/src/main/geonetwork-ui/package.json");
```

Comment:
Wait, that won't work in a deployed WAR scenario; this file is available in the source tree of the project but will not be copied over to the JAR right?

I think we need to do `FileUtils.getFileFromJar(DEFAULT_FILES_PATH + "package.json");` and make sure in the `pom.xml` that the package.json is also copied over when building the JAR of the plugin